catalog/imaging/imaging-actionprovider-chip/src/main/java/org/codice/alliance/imaging/chip/actionprovider/ImagingChipActionProvider.java

```diff
@@ -56,9 +62,8 @@
             URL url;
             String chipPath = null;
             try {
-                chipPath =
-                        SystemBaseUrl.getBaseUrl() + PATH + "?id=" + metacard.getId() + "&source="
-                                + metacard.getSourceId();
+                chipPath = SystemBaseUrl.getBaseUrl() + PATH + "?id=" + metacard.getId()
```

Comment:
Outside the scope of your changes, but the return value of this method should probably be changed to 
`return Collections.singletonList(new ActionImpl(getId(), TITLE, DESCRIPTION, url));`